event-handler/event_handler_test.go

```diff
@@ -241,7 +241,7 @@ func (s *EventHandlerSuite) TestEvents() {
 	require.Contains(t, evt, `"user":"`+s.userNames.plugin+`"`)
 	require.Contains(t, evt, `"roles":["access-event-handler"]`)
 
-	err = s.ruler().CreateUser(s.Context(), &types.UserV2{
+	_, err = s.ruler().CreateUser(s.Context(), &types.UserV2{
```

Comment:
This must be reverted when we start using the `branch/v14` version of the Teleport API

Reply:
👍